packages/core/src/endowmentsToolkit.js

```diff
@@ -1,12 +1,26 @@
-// the contents of this file will be copied into the prelude template
-// this module has been written so that it required directly or copied and added to the template with a small wrapper
```

Comment:
(could) I see this was moved into the comment below, but I'd leave at least a short warning at the top